app/models/project_media_project.rb

```diff
@@ -180,6 +181,16 @@ def self.bulk_update(input_ids, updates, team)
     { team: team, project: project, project_was: project_was, check_search_project_was: project_was&.check_search_project, ids: pm_graphql_ids }
   end
 
+  def slack_channel(event)
+    slack_events = self.project.setting(:slack_events)
```

Comment:
Will this always return an array? The line below assumes so.

Reply:
Yes and I should validate the slack settings to be sure it's an array before saving